src/fileops/davmeta.cpp

```diff
@@ -174,7 +213,12 @@ dav_ssize_t getStatInfo(Context & c, const Uri & url, const RequestParams * p,
             ret = dav_stat_mapper_webdav(c, &params, url, st_info);
             break;
         default:
-            ret = dav_stat_mapper_http(c, &params, url, st_info);
+            if (url.queryParamExists("AWSAccessKeyId") && url.queryParamExists("Signature")) {
```

Comment:
Hi Oliver - thanks for the PR! The line above will only detect URLs signed with v2 signatures though, not v4.

The equivalent parameters for v4 are "X-Amz-Credential" and "X-Amz-Signature". Let me know if you'd prefer me to merge your PR and fix it, or you'd rather update the PR.